code/game/gamemodes/malfunction/newmalf_ability_trees/HELPERS.dm

```diff
@@ -171,11 +171,37 @@
 /proc/get_unhacked_apcs(var/mob/living/silicon/ai/user)
 	var/list/H = list()
 	for(var/obj/machinery/power/apc/A in machines)
-		if(A.hacker && A.hacker == user)
+		if(!A.hacker && A.hacker != user)
```

Comment:
This still :D 

It doesn't make sense. `!A.hacker` and `A.hacker != user` can never be true at the same time. So this AND condition will always be false.

Are you sure you even need to modify this proc...? An unhacked APC (this one gets those) should be `if (A.hacker && A.hacker == user) => continue` Because you want to skip the APCs that are hacked.